docs/conf.py

```diff
@@ -263,6 +263,7 @@
 # Extra intersphinx in addition to what is already in sphinx-astropy
 intersphinx_mapping.update({  # noqa: F405
     'glueviz': ('https://docs.glueviz.org/en/stable/', None),
+    'astroquery': ('https://astroquery.readthedocs.io/en/stable/', None),
```

Comment:
Given upstream service provider stuff can change faster than release, would `latest` be safer here? I dunno.

Reply:
I don't know either, I chose stable since all other intersphinx links were stable.

Reply:
glue points to latest 🤷‍♀️